README.md

```diff
@@ -40,6 +40,7 @@ Current version 1.0.8
   `kotlinVersion`
   plugin
 * `kotlincOptions`: options to pass to the kotlin compiler
+* `definedTests in Test := KotlinTest.kotlinTests.value`: specifies kotlin test classes
```

Comment:
this seems unnecessary to document unless there's a specific user-need that they would want to override this?

Reply:
Yes, this line doesn't have sense because I have added in the kotlinSettings. I will remove it. 💯